Feature/Education.feature

```diff
@@ -0,0 +1,20 @@
+﻿Feature: Education
+User wants to add education details into Profile Details
+Acceptance Criteria
+User is able to add the education details
+
+@mytag
+ Scenario: 01 Test Add Education sucessfully
+	Given User enter UniversityName as "Delhi University"
+	And   Select CountryOfUniversity as "India"
+	And   Select Title as "B.Tech"
+	And   Enter Degree as "Computer Science"
+	And   Select YearOfGreaduation as "2012"  
+	When  Clicks on Add button on AddEducation page
+	Then  Education should be added to the  profile. The Alert message "Education has been added" is displayed.

```

Comment:
You are testing the UI of the profile page right?

Does testing the alert message tells you that you can see and confirm whatever you entered was indeed added

Furthermore, I think you should be focusing and testing/asserting the details that you have added instead of alert message. Imagine if the alert massage is changed, your test case will break.

Please review all your Then keywords